src/store/asana/index.ts

```diff
@@ -408,15 +411,64 @@ export const useAsanaStore = defineStore("asana", {
     DELETE_STORY(gid: string): void {
       const prefStore = usePrefStore();
       if (gid) {
-        this.ADD_ACTION(
-          "deleting story",
-          async () => {
-            await asanaClient?.stories?.delete(gid);
-            if (prefStore?.taskEditorSectionIdAndTask?.task) {
-              prefStore.taskEditorSectionIdAndTask.task.stories = prefStore.taskEditorSectionIdAndTask.task.stories.filter((story) => story.gid !== gid);
-            }
+        this.ADD_ACTION("deleting story", async () => {
+          await asanaClient?.stories?.delete(gid);
+          if (prefStore?.taskEditorSectionIdAndTask?.task) {
+            prefStore.taskEditorSectionIdAndTask.task.stories =
+              prefStore.taskEditorSectionIdAndTask.task.stories.filter(
+                (story) => story.gid !== gid
+              );
           }
-        );
+        });
+      }
+    },
+
+    ADD_PROJECT_TO_TASK(task: string, project: string, section?: string) {
```

Comment:
Are the parameters the "task" or the id? If the latter, let's update their names to suffix with Id